packages/web-components/src/components/va-crisis-line-modal/test/va-crisis-line-modal.e2e.ts

```diff
@@ -87,6 +87,7 @@ describe('va-crisis-line-modal', () => {
       'va-crisis-line-modal >>> button.va-overlay-trigger',
     );
     await triggerButton.click();
+    await page.waitForChanges();
```

Comment:
@Andrew565 Sorry, I missed this in my last review. Do you need this still?

Reply:
Probably not, thanks for catching.